src/cljs/main/broadfcui/page/workspace/create.cljs

```diff
@@ -93,8 +95,11 @@
 
 (react/defc Button
   {:render
-   (fn [{:keys [props]}]
+   (fn [{:keys [props state]}]
      [:div {:style {:display "inline"}}
+      (when (:modal? @state)
+        [CreateDialog (merge (select-keys props [:billing-projects])
+                             {:dismiss #(swap! state dissoc :modal?)})])
```

Comment:
Besides slightly improved functionality, this is the primary difference: We can now use modals similarly to normal React elements.